### PR TITLE
Assingment2

### PR DESCRIPTION
**Pull request by MiguelERuiz**

### CODE REVIEW

File: Assignment2/MiguelERuiz-MiguelEmilioRuizNieto.ttl [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix : <http://www.ontogrid.net/StickyNote#> .
```
Comment (ocorcho) [inline Assignment2/MiguelERuiz-MiguelEmilioRuizNieto.ttl]: This is not right syntax

File: Assignment2/MiguelERuiz-MiguelEmilioRuizNieto.ttl [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+:Class01 dc:includes :Sensor029, :Computer101 .
+
+:Sensor029 dc:hasMeasurement :Measurement8401 .
+
+:Measurement8401
+    dc:hasTemperature "29" ;
```
Comment (ocorcho) [inline Assignment2/MiguelERuiz-MiguelEmilioRuizNieto.ttl]: Make sure that you add a datatype to this and the following line

**Comment by MiguelERuiz** [comment]
Hi Oscar,

I did the changes you requested, please tell me if you see them or not.

**Comment by ocorcho** [comment]
Done.

**Pull request merged**